robottelo/ui/rhai.py

```diff
@@ -0,0 +1,14 @@
+# -*- encoding: utf-8 -*-
+""" Implements methods for RHAI"""
+
+from robottelo.ui.base import Base
+from robottelo.ui.locators import locators
+
+
+class RHAI(Base):
+    def view_registered_systems(self):
+        """To view the number of registered systems"""
+        result = self.wait_until_element(
+            locators['insights.registered_systems']
+        ).text
+        return result
```

Comment:
Elyzer, this is a initial test case where I check that if the provisioned VM(registered) shows up in the count of number of systems in the UI or not. I would we writing another test case for verifying form the list or may be change this later as and when required, but IMO this test case is sufficient as a start. What do you think?